bnelson777/id.ly
# IL-115: Implement android url scheme

Uses a redirection link set up by Joe since deep links aren't clickable on android. 

To test:
-Create a new message, email to yourself and click the link. The app should open and you should be taken to the decrypt screen with the information filled in. Click "decrypt" and message should be added to your inbox.
